src/fusiontensor/fusiontensor.jl

```diff
@@ -209,6 +214,47 @@ function FusionTensor(
   return FusionTensor(x, tuplemortar((codomain_legs, domain_legs)))
 end
 
+function FusionTensor{T}(
+  x,
+  codomain_legs::Tuple{Vararg{AbstractGradedUnitRange}},
+  domain_legs::Tuple{Vararg{AbstractGradedUnitRange}},
+) where {T}
+  return FusionTensor{T}(x, tuplemortar((codomain_legs, domain_legs)))
+end
+
+# specific constructors
+function Base.zeros(::Type{T}, fta::FusionTensorAxes) where {T}
+  ft = FusionTensor{T}(undef, fta)
+  map(m -> fill!(m, zero(T)), eachstoredblock(data_matrix(ft)))
```

Comment:
```suggestion
  foreach(m -> fill!(m, zero(T)), eachstoredblock(data_matrix(ft)))
```
since you don't need to collect the results.